Bug During Compilation

Controller: STM32L051C8

Issue: 17 | #if !defined(CUSTOM_PERIPHERAL_PINS)
      | 
Using library SrcWrapper at version 1.0.1 in folder: C:\Users\omkar\AppData\Local\Arduino15\packages\STMicroelectronics\hardware\stm32\2.0.0\libraries\SrcWrapper 
exit status 1
processing.app.debug.RunnerException
	at cc.arduino.Compiler.lambda$callArduinoBuilder$3(Compiler.java:309)
	at processing.app.debug.MessageSiphon.run(MessageSiphon.java:96)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at cc.arduino.Compiler.message(Compiler.java:525)
	at cc.arduino.i18n.I18NAwareMessageConsumer.message(I18NAwareMessageConsumer.java:80)
	at cc.arduino.MessageConsumerOutputStream.flush(MessageConsumerOutputStream.java:71)
	at cc.arduino.MessageConsumerOutputStream.write(MessageConsumerOutputStream.java:54)
	at java.io.OutputStream.write(OutputStream.java:75)
	at cc.arduino.Compiler.lambda$callArduinoBuilder$3(Compiler.java:307)
	... 2 more

Code:
void setup() {
  // put your setup code here, to run once:

}

void loop() {
  // put your main code here, to run repeatedly:

}

Hi @srjtmitra5 
Thanks for the report but it misses the main information of the error 😉 :
```log
packages\STMicroelectronics\hardware\stm32\2.0.0\variants\STM32L0xx\L051C(6-8)(T-U)\PeripheralPins_RHF76_052.c:17: error: unterminated #if
   17 | #if !defined(CUSTOM_PERIPHERAL_PINS)
      | 
```

--- a/variants/STM32L0xx/L051C(6-8)(T-U)/PeripheralPins_RHF76_052.c
+++ b/variants/STM32L0xx/L051C(6-8)(T-U)/PeripheralPins_RHF76_052.c
@@ -14,7 +14,7 @@
  * Automatically generated from STM32L051C(6-8)Tx.xml, STM32L051C(6-8)Ux.xml
  * CubeMX DB release 6.0.20
  */
-#if !defined(CUSTOM_PERIPHERAL_PINS)
+#if defined(ARDUINO_RHF76_052)
 #include "Arduino.h"
 #include "PeripheralPins.h"
 
@@ -192,3 +192,4 @@ WEAK const PinMap PinMap_SPI_SSEL[] = {
 //*** No USB ***
 
 //*** No SD ***
+#endif /* ARDUINO_RHF76_052 */

